Split/Network/Streaming/SyncManager.swift

```diff
@@ -73,11 +73,14 @@ class DefaultSyncManager: SyncManager {
             isPollingEnabled.set(false)
             Logger.i("Polling disabled")
 
-        case .pushSubsystemDown:
-            Logger.d("Push Subsystem Down event message received.")
+        case .pushSubsystemPaused:
+            Logger.d("Push Subsystem Paused event message received.")
             reconnectStreamingTimer.cancel()
             enablePolling()
-            pushNotificationManager.stop()
+
+        case .pushSubsystemDown:
```

Comment:
`pushSubsystemDown` shouldn't stop streaming, because this event is also emitted internally when occupancy publishers is 0.

I think you should go back to your previous approach where:
- `pushSubsystemDown` was emitted on `STREAMING_PAUSED` control notification, and when occupancy publishers is equal to 0. `pushSubsystemDown` should have the logic of the new `pushSubsystemPaused` event (thus `pushSubsystemPaused` constant is not necessary).
- `pushDisabled` was emitted on `STREAMING_DISABLED`, and it has the same logic as a non retryable error.